builds/android/AndroidManifest.xml

```diff
@@ -4,8 +4,8 @@
 -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
       package="org.easyrpg.player"
-      android:versionCode="1"
-      android:versionName="1.0"
+      android:versionCode="2533"
+      android:versionName="0.2.2.70"
```

Comment:
That wasnt an incorrect merge. This happened because he used the build script we use in Jenkins. This replaces the versionCode and versionName.